### DISCUSSION TITLE
GitHub student verification failed for Romanian high school card

### DISCUSSION THREAD

**Discussion by AlexVasiluta** [community/community#52712] [GitHub Education]
Hello everyone!

I am in my 3rd year of high school in Romania and I received a message that my GitHub Student Discount was about to expire. Until recently (including middle school, but also in high school), my school card was considered enough for verification. It should be mentioned that Romanian school card information is always handwritten, so it might mess up automatic detection systems that may have been recently implemented.

The specific message that is returned in the rejection is as follows:
> Unfortunately, we weren't able to approve your educational discount request for @AlexVasiluta.
> What happened?
> The item you uploaded is insufficient to demonstrate your current academic status.
> Please see suggestions.
> Suggested proof to expedite your approval:
> -  You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.

I have updated my profile with my full name, location, school, and a short bio, but it seems to be in vain.

I would like to add the fact that I have an educational email linked to my account (it was selected as the institutional email while sending the discount request, and the school is also recognized), and that the name of the school (along with location and phone contact information) appears on the school card, albeit in a handwritten manner.

My question is, do I now need other documents, like a school transcript (which may a bit harder to get here without a more "valid" reason in their eyes), for validation?

**Comment by mrbot23** [comment]
Have you had this issue solved? I'm trying to sign up for GitHub Education but I have the same issue with it being handwritten.

**Reply by AlexVasiluta** [reply]
Hello!
I could not get it to work again since then. Because i'm now in a terminal year i got a copy of my matriculation sheet and used that as proof to get the benefits, I don't think there are other ways.
Hope you have more luck in the future, though!
